### PR TITLE
KevS-SK1480-Creation-Manual-installation-Grav-on-WebHosting-TRANSLATIONS

### PR DESCRIPTION
**Pull request by Kazarus4**
Translations still in progress.

DE, EN, ES are already done.

### CODE REVIEW

**Comment by Kazarus4** [comment]
All translations are done.

Need proof by @tcpdumpfbacke for DE and EN subs and by @Y0Coss  for latin and PL subs.

DO NOT MERGE AFTER THAT BECAUSE I MUST UPDATE ANOTHER GUIDE IN THE SAME TIME.

File: pages/web_cloud/web_hosting/cms_manual_installation_grav/guide.es-es.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+> [!primary]
+> Esta traducción ha sido generada de forma automática por nuestro partner SYSTRAN. En algunos casos puede contener términos imprecisos, como en las etiquetas de los botones o los detalles técnicos. En caso de duda, le recomendamos que consulte la versión inglesa o francesa de la guía. Si quiere ayudarnos a mejorar esta traducción, por favor, utilice el botón "Contribuir" de esta página.
+>
+
+## Objetivo
+
+El **CMS** (**C**ontent **M**anagement **S**ystem) Grav permite desarrollar rápidamente sitios web. Diseñado para una gestión optimizada del contenido a través de archivos Markdown, Grav se adapta perfectamente a la creación de sitios web personales o proyectos para pequeñas empresas, sin comprometer la calidad o la personalización.
+
+**Descruba cómo instalar manualmente el CMS Grav en un alojamiento web de OVHcloud.**
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/web_hosting/cms_manual_installation_grav/guide.es-es.md]:
```suggestion
**Descubra cómo instalar manualmente el CMS Grav en un alojamiento web de OVHcloud.**
```

File: pages/web_cloud/web_hosting/cms_manual_installation_grav/guide.es-us.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+> [!primary]
+> Esta traducción ha sido generada de forma automática por nuestro partner SYSTRAN. En algunos casos puede contener términos imprecisos, como en las etiquetas de los botones o los detalles técnicos. En caso de duda, le recomendamos que consulte la versión inglesa o francesa de la guía. Si quiere ayudarnos a mejorar esta traducción, por favor, utilice el botón "Contribuir" de esta página.
+>
+
+## Objetivo
+
+El **CMS** (**C**ontent **M**anagement **S**ystem) Grav permite desarrollar rápidamente sitios web. Diseñado para una gestión optimizada del contenido a través de archivos Markdown, Grav se adapta perfectamente a la creación de sitios web personales o proyectos para pequeñas empresas, sin comprometer la calidad o la personalización.
+
+**Descruba cómo instalar manualmente el CMS Grav en un alojamiento web de OVHcloud.**
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/web_hosting/cms_manual_installation_grav/guide.es-us.md]:
```suggestion
**Descubra cómo instalar manualmente el CMS Grav en un alojamiento web de OVHcloud.**
```

**Review by OVH-Team-Guides — APPROVED** [review]

**Comment by Kazarus4** [comment]
DO NOT MERGE WHILE https://github.com/ovh/docs/pull/6042 NOT READY TO MERGE, SPIP AND TYPO WILL NOT TRANSLATED TOO

**Review by bs-ovh — APPROVED** [review]

**Comment by Kazarus4** [comment]
READY to MERGE when #6050 will be proof for DE and EN, and EN corrections duplicated.

**Pull request merged**